core/gather/navigation-runner.js

```diff
@@ -187,9 +187,15 @@ async function _computeNavigationResult(
 
     /** @type {Partial<LH.GathererArtifacts>} */
     const artifacts = {};
-    const pageLoadErrorId = `pageLoadError-${navigationContext.navigation.id}`;
-    if (debugData.devtoolsLog) artifacts.devtoolsLogs = {[pageLoadErrorId]: debugData.devtoolsLog};
-    if (debugData.trace) artifacts.traces = {[pageLoadErrorId]: debugData.trace};
+    const navigationId = navigationContext.navigation.id;
+    if (debugData.devtoolsLog) {
+      artifacts.DevtoolsLog = debugData.devtoolsLog;
+      artifacts.devtoolsLogs = {[navigationId]: debugData.devtoolsLog};
```

Comment:
Asset saver doesn't have any logic to load a `pageLoadError-defaultPass.trace.json` trace into `artifacts.Traces`. That logic is reserved for a file named `defaultPass.trace.json`.

For this PR we should avoid using anything other than `defaultPass` as a key for the compat `devtoolsLogs`/`traces` artifacts.

Reply:
> Asset saver doesn't have any logic to load a `pageLoadError-defaultPass.trace.json` trace into `artifacts.Traces`. That logic is reserved for a file named `defaultPass.trace.json`.

I can't imagine there'd ever be a time you'd want to load an pageLoadError trace (it would go through the same `runner` code and still not produce any audit results), but there is an advantage to _saving_ them with a different filename than the default.

Reply:
If we are going to make `devtoolsLogs`/`traces` true compat artifacts then we can't do anything with them that's not possible to do with `artifacts.DevtoolsLog`/`artifacts.Trace`. Keying the artifact files differently for error state is one of those things.

Reply:
> If we are going to make `devtoolsLogs`/`traces` true compat artifacts then we can't do anything with them that's not possible to do with `artifacts.DevtoolsLog`/`artifacts.Trace`.

But why not? :)

If the guide is the principle of least surprise, considering these can only be used for debugging via `-A` and never in an audit, I think it's more important to avoid anyone confusing the trace/devtoolslog of `chrome-error://chromewebdata/` with a trace/devtoolslog of the `example.com` they thought they loaded.

It does seem like your `artifacts.DevtoolsLogError`/`artifacts.TraceError` idea would sidestep the conflict between the two goals, however.

Reply:
I gave `artifacts.DevtoolsLogError`/`artifacts.TraceError` a go. Changes to `runner.js` are no longer necessary for this PR but I still think they are good.